include/RED4ext/StringView.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <RED4ext/Common.hpp>
+
+#include <string_view>
+
+namespace RED4ext
+{
+struct CString;
+
+#pragma pack(push, 4)
```

Comment:
Is this really aligned to 4 bytes?

Reply:
Not sure, but Visual Studio was putting in padding - should I use a different way of making sure VS does not insert padding?

Reply:
No, that's a good way I also use. But I want curious if the game align this struct to 4 instead of the default of 8.

Reply:
I think it's 4 aligned. I've seen one example of `StringView` array with element size of `12`.